### PR TITLE
expected modal, solo and mute behaviour

### PR DESCRIPTION
**Pull request by muganwas**

### CODE REVIEW

**Review by choshun — COMMENTED** [review]
What is the 
> expected modal, solo and mute behaviour
?

File: src/components/play/layer-settings/LayerSettings.js [hunk 1]
```diff
@@ -459,6 +459,12 @@ class LayerSettings extends Component {
         this.showDeleteClearPopupButton = React.createRef()
         this.layerPopupButton = React.createRef()
         this.volumePopupButton = React.createRef()
+
+        this.muteToggle = React.createRef()
```
Comment (choshun) [inline src/components/play/layer-settings/LayerSettings.js:463]: Why did you need to add this? Could this be accomplished with a prop like 'isMuted'?

Comment (muganwas) [reply src/components/play/layer-settings/LayerSettings.js:463]: this is a ref, works like an id, when used to getElement

Comment (muganwas) [reply src/components/play/layer-settings/LayerSettings.js:463]: I could add Ref to the end of all the refs eg. muteToggleRef, I guess that would be more readable.

Comment (choshun) [reply src/components/play/layer-settings/LayerSettings.js:463]: Oh, just more wondering why you need this at all

I'd think the mute toggle would affect the state of the layer, then you could use that state to inform other stuff in the layer, not really sure why you need to ref the DOM

Comment (choshun) [reply src/components/play/layer-settings/LayerSettings.js:463]: Talked to @muganwas and the reason is because we need to only affect state if the event target is something specific. Good reason to use this!

**Review by choshun — APPROVED** [review]
LGTM!

This is really coming along - looks very slick, great job
